WtWSMS/common/culture/creation_names/BP_ck2_custom_names.txt

```diff
@@ -0,0 +1,238 @@
+﻿#The most glorious of all cultures, East Celtic
+east_celtic = {
+	trigger = {
+		has_valid_east_celtic_ancestors_trigger = {
+			CULTURE_1 = scope:culture
+			CULTURE_2 = scope:other_culture
+		}
+	}
+	hybrid = yes
+}
+##### THIS SECTION BASED ON CK2/WtWSMS/events/WtWSMS_culture_conversion_events.txt
+#Note that since these *aren't* county events in CK3, we don't do the religion checks wrt counties
+#ROM.1000: Skipped (Duplicate French Creation)
+#ROM.1100: Skipped (Duplicate Occitan Creation)
+#ROM.1150: Skipped (Visigoth back to Romano-Gallic)
+gothi = { #ROM.2000; Gothi
+	trigger = {
+		root = {
+			culture = culture:visigothic
+			faith.religious_head = faith:nicene.religious_head
+			capital_county = { any_county_province = { geographical_region = world_europe_west_iberia } }
+		}
+		scope:other_culture = { this = culture:romano_hispanic }
+	}
+	hybrid = yes
+}
```

Comment:
Gothi had precisely been removed because it was considered inaccurate and bizarre to have an additional layer in Iberia when it was done nowhere else. Not convinced it should be added back in

Reply:
Well, it wasn't clear in the CK2 event space what was actually removed vs temporarily removed. But mechanicalistically, everything in `creation_names/` only influences the name/collective noun/prefix of a culture; it doesn't impact the name lists (which is why I link the name lists with the linguistic drift events).

So a better question is whether or not the name itself should be removed; to be explicit, I don't have strong feelings one way or the other, just need to be clear about what `common/creation_names/` does.

Reply:
Just to be clear, none of the Romance ones in the old file `WtWSMS_culture_conversion_events` were in use unless you made the explicit choice as the player to go back to the old system. But that is largely what you did, so no problems, just the name Gothi which I don't think should be kept, especially given the new mechanics.

Reply:
Alright, I'll make those corrections today.